### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ghost**
minor escape symbols and markup fix

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: cmd/gaiacli/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 This branch uses non-canonical branch of cosmos-sdk. Before building, run `go mod vendor` on the root directory to retrive the dependencies. To build:
 
 ```shell
-git clone git@github.com:cosmos/gaia
+git clone https://github.com/:cosmos/gaia
```
Comment (alexanderbez) [inline cmd/gaiacli/README.md:10]:
```suggestion
git clone https://github.com/cosmos/gaia.git
```

**Review by alexanderbez — CHANGES_REQUESTED** [review]

**Review by alexanderbez — APPROVED** [review]
ACK

**Comment by alexanderbez** [comment]
@everstake mind rebasing/fixing conflicts please?

**Comment by fedekunze** [comment]
closing as outdated. Check out the new PR for the alpha implementation #172

**Pull request closed without merging**